packages/core/test/utils.spec.js

```diff
@@ -0,0 +1,309 @@
+                scope: null,
+                color: null,
+            };
+            expect(utils.lang('file.this_is_not_a_file')).toEqual(expected);
+        });
+    });
+
+    describe('.titlize()', () => {
+        it('returns titlized string', () => {
+            expect(utils.titlize('not-a-title')).toEqual('Not A Title');
+        });
+    });
+
+    describe('.slugify()', () => {
+        it('returns slugified string', () => {
+            expect(utils.slugify('Not A Slug')).toEqual('not-a-slug');
+        });
+    });
+
+    describe('.escapeForRegexp()', () => {
+        it('returns escaped string', () => {
+            expect(utils.escapeForRegexp('$handle')).toEqual('\\$handle');
+        });
+    });
+
+    describe('.parseArgv()', () => {
+        it('returns correct data for command with arguments and options', async () => {
+            const expected = {
+                args: ['bar'],
+                command: 'foo',
+                opts: {
+                    option: 'value',
+                },
+            };
+            await mockArgv(['foo', 'bar', '--option', 'value'], () => {
+                expect(utils.parseArgv()).toEqual(expected);
+            });
+        });
+
+        it('returns correct data without command', async () => {
+            const expected = {
+                args: [],
+                command: null,
+                opts: {},
+            };
+            await mockArgv([], () => {
+                expect(utils.parseArgv()).toEqual(expected);
+            });
+        });
+    });
+
+    describe('.stringify()', () => {
+        it('returns stringified object', () => {
+            const object = {
+                buffer: Buffer.from('buffer'),
+                func: function () {},
+                plainObject: {},
+            };
+            const expected = `{
+    "buffer": "<Buffer>",
+    "func": "<Function>",
+    "plainObject": "{}"
+}`;
+            expect(utils.stringify(object)).toEqual(expected);
+        });
+
+        it('allows custom indent', () => {
+            const object = {
+                buffer: Buffer.from('buffer'),
+                func: () => {},
+                plainObject: {},
+            };
+            const expected = `{
+  "buffer": "<Buffer>",
+  "func": "<Function>",
+  "plainObject": "{}"
+}`;
+            expect(utils.stringify(object, 2)).toEqual(expected);
+        });
+    });
+
+    describe('.fileExistsSync()', () => {
+        it('returns true if file is accessible', () => {
+            mock({
+                'path/to/accessible/file': 'can access',
+            });
+
+            expect(utils.fileExistsSync('path/to/accessible/file')).toBe(true);
+
+            mock.restore();
+        });
+    });
+
+    describe('.md5()', () => {
+        it('returns same hash for same string', () => {
+            const first = utils.md5('string');
+            const second = utils.md5('string');
+            expect(first).toEqual(second);
```

Comment:
Worth checking it’s a string of some length or at least not empty/null/undefined?

Reply:
Added an actual md5 value assertion.